src/markdown-pages/docs/cli-command-reference/runs-reset-cancel.md

```diff
@@ -0,0 +1,46 @@
+---
+path: "/docs/cli-command-reference/runs-reset-cancel"
+title: "Retrying and deleting tests"
+---
+
+
+Sometimes tests can become stuck in a loop and fail to finish running, for example, due to a lack of available resources, an environmental problem, or a timeout. You can retry a test to run again by using the `runs reset` command. If the test continues to fail to finish running, you can use the `runs cancel` command to delete the test. 
```

Comment:
At the end of this sentence, I believe '`runs cancel` command to abandon the test' would be better, again to avoid confusion about deleting from RAS.